.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build Docker Image
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [v1.14]
+        os: [alpine]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          file: ./${{ matrix.version }}/${{ matrix.os }}/Dockerfile
+          context: ./${{ matrix.version }}/${{ matrix.os }}
+          platforms: linux/amd64,linux/arm64
+          push: false
```

Comment:
I'd be tempted to load and upload an artifact in case people want to pull and test it?

Reply:
We could gate a push on it being a ref in this repo, I do mean to actually gate most of the CI so it either doesn't run or works on forks. I was also considering removing the arm image from CI to speed up the feedback loop.